lesson26/login.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <title>TerraceTechフロントエンドエンジニア養成所</title>
+  <link rel="stylesheet" href="css/reset.css" type="text/css" />
+  <link rel="stylesheet" href="css/common.css" type="text/css" />
+  <link rel="stylesheet" href="css/login.css" type="text/css" />
+  <script>localStorage.getItem("token") && (window.location.href = "./news.html");
+  </script>
+  <script src="js/login.js"></script>
```

Comment:
There is no error in codesandbox, but when I looked at it in my local environment, I could see the error.
I think the problem is the timing at which the script is loaded.
<a href="https://gyazo.com/2fe6362b582413e883c7c9c6d7bc656b"><img src="https://i.gyazo.com/2fe6362b582413e883c7c9c6d7bc656b.png" alt="Image from Gyazo" width="739"/></a>

You can try using `defer`
```html
    <script src="js/login.js" defer></script>

Reply:
Thank you, I didn't realize this error.
And also I learn about defer attribute this time. It's thanks to you!
[d32ab18](https://github.com/yuka830/js-lessons/pull/31/commits/d32ab18dfcf61f60d3e529dd4c39a9c52dee3686)